rover_auxiliary/src/ipcamera_api.py

```diff
@@ -0,0 +1,322 @@
+import logging, requests, signal, string
```

Comment:
I would change the logging to use the default ROS logger, maybe take one as argument at init